packages/react/roving-focus/src/RovingFocusGroup.tsx

```diff
@@ -201,14 +201,22 @@ const ITEM_NAME = 'RovingFocusGroupItem';
 type RovingFocusItemElement = React.ElementRef<typeof Primitive.span>;
 type PrimitiveSpanProps = Radix.ComponentPropsWithoutRef<typeof Primitive.span>;
 interface RovingFocusItemProps extends PrimitiveSpanProps {
+  tabStopId?: string;
   focusable?: boolean;
   active?: boolean;
 }
 
 const RovingFocusGroupItem = React.forwardRef<RovingFocusItemElement, RovingFocusItemProps>(
   (props: ScopedProps<RovingFocusItemProps>, forwardedRef) => {
-    const { __scopeRovingFocusGroup, focusable = true, active = false, ...itemProps } = props;
-    const id = useId();
+    const {
+      __scopeRovingFocusGroup,
+      focusable = true,
+      active = false,
+      tabStopId,
+      ...itemProps
+    } = props;
+    const autoId = useId();
+    const id = tabStopId || autoId;
```

Comment:
I needed to control the tab stop as it's possible for the trigger to never receive focus in some cases.

I may have missed something but I couldn't find a way to provide my own ids, this made the existing `defaultCurrentTabStopId` prop confusing / useless?